src/stories/Interactive.stories.mdx

```diff
@@ -55,10 +61,31 @@ A TR to represent a draggable person, etc in a list
 <Canvas>
   <Story name="Person TR">
     {() => {
-      const michiel = UI.rdf.namedNode('https://michielbdejong.inrupt.net/profile/card#me')
+      const michiel = UI.rdf.namedNode(
+        'https://michielbdejong.inrupt.net/profile/card#me'
+      )
       return UI.widgets.personTR(document, null, michiel)
-    }
-    }
+    }}
+  </Story>
+</Canvas>
+
+## renderAsDIV
+
+A Div to represent a draggable person, etc in a list
+
+[API documentation](https://solid.github.io/solid-ui/Documentation/api/modules/_widgets_buttons_.html#persontr)
+
+<Canvas>
```

Comment:
**praise** Thanks for adding a story.

**issue** It does not look well aligend, is this only in storybook or a general problem with the component?

![](https://user-images.githubusercontent.com/340725/123448937-7e647900-d5db-11eb-84a7-289cfc269921.png)